recipes/resource_pool/all/test_package/conanfile.py

```diff
@@ -0,0 +1,14 @@
+from conans import ConanFile, CMake
+
+
+class ResourcePoolTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake_find_package"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        pass # Building alone is sufficient
```

Comment:
better to execute something to verify the app is able to run and there are no runtime errors

Reply:
Sorry for that, I actually chose this example from the upstream repository mostly because it failed to build if certain ASIO defines were missing, so I did not really pay attention to what it does.

It actually performs file IO and leaves the build directory in a non-consistent state and signals errors only in output, so I'd refrain from using it as-is. I'll change it accordingly.